randhound/protocol/utils.go

```diff
@@ -21,9 +21,11 @@ func (rh *RandHound) newSession(nodes int, groups int, purpose string, timestamp
 	var err error
 
 	if timestamp.IsZero() {
-		timestamp = time.Now()
+		timestamp = time.Now().UTC()
```

Comment:
I think doing `time.Unix()` https://golang.org/pkg/time/#Time.Unix would be better here. Then you don't have to worry about UTC or others.